torch/testing/_internal/common_utils.py

```diff
@@ -86,7 +86,7 @@
     Pair,
     TensorLikePair,
 )
-from torch.testing._comparison import assert_equal as assert_equal
+from torch.testing._comparison import are_equal as are_equal
```

Comment:
Just curious, why `as` is needed here?

Reply:
I have no idea. This syntax was introduced in #87619. cc @bschnurr @kit1980 I just kept it.

Reply:
`torch/testing/__init__.py` is curated and contains stuff that should be public:

https://github.com/pytorch/pytorch/blob/97e47a52b8872873533abc22f93a919222799864/torch/testing/__init__.py#L1-L3

Can we remove the `as` aliasing there?